service.subtitles.pipocas/service.py

```diff
@@ -0,0 +1,604 @@
+            if _parentfolder == '0':
+                if re.search(release_pattern, dirsearch[-1], re.IGNORECASE):
+                    _parentfolder = '1'
+                else:
+                    _parentfolder = '2'
+            if _parentfolder == '1':
+                if re.search(dirsearch[-1], desc, re.IGNORECASE):
+                    sync = True
+                else:
+                    sync = False
+            if _parentfolder == '2':
+                if (searchstring_notclean != ""):
+                    sync = False
+                    if str.lower(searchstring_notclean) in str.lower(desc):
+                        sync = True
+                else:
+                    if (str.lower(dirsearch_check[-1]) == "rar") or (str.lower(dirsearch_check[-1]) == "cd1") or (str.lower(dirsearch_check[-1]) == "cd2"):
+                        sync = False
+                        if len(dirsearch) > 1 and dirsearch[1] != '':
+                            if re.search(filesearch[1][:len(filesearch[1])-4], desc, re.IGNORECASE) or re.search(dirsearch[-2], desc, re.IGNORECASE):
+                                sync = True
+                        else:
+                            if re.search(filesearch[1][:len(filesearch[1])-4], desc, re.IGNORECASE):
+                                sync = True
+                    else:
+                        sync = False
+                        if len(dirsearch) > 1 and dirsearch[1] != '':
+                            if re.search(filesearch[1][:len(filesearch[1])-4], desc) or re.search(dirsearch[-1], desc, re.IGNORECASE):
+                                sync = True
+                        else:
+                            if re.search(filesearch[1][:len(filesearch[1])-4], desc, re.IGNORECASE):
+                                sync = True
+            filename = filename + "  " + "hits: " + hits + " uploader: " + uploader
+            subtitles_list.append({'rating': str(downloads), 'filename': filename, 'hits': hits, 'desc': desc,
+                                   'sync': sync, 'id': id, 'language_short': languageshort, 'language_name': languagelong})
+        page = page + 1
+        if languageshort == "pt":
+            url = main_url + "legendas?t=rel&l=portugues&page=" + str(page) + "&s=" + searchstring
+        elif languageshort == "pb":
+            url = main_url + "legendas?t=rel&l=brasileiro&page=" + str(page) + "&s=" + searchstring
+        elif languageshort == "es":
+            url = main_url + "legendas?t=rel&l=espanhol&page=" + str(page) + "&s=" + searchstring
+        elif languageshort == "en":
+            url = main_url + "legendas?t=rel&l=ingles&page=" + str(page) + "&s=" + searchstring
+        else:
+            url = main_url + "home"
+        content = sessionPipocasTv.get(url)
+
+    # Bubble sort, to put syncs on top
+    subtitles_list = bubbleSort(subtitles_list)
+    return subtitles_list
+
+
+def append_subtitle(item):
+    
+    listitem = xbmcgui.ListItem(
```

Comment:
in all your listitems please use offscreen=True. Reason: https://xbmc.github.io/docs.kodi.tv/master/kodi-base/d8/d29/group__python__xbmcgui__listitem.html#ga15bcb930a4f91dd97f8ed7c633a20618

Reply:
done